packages/mesos/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "13b7c1143b2ced464d4fca0bb381b158ffaed8ad",
+    "ref": "08f0b6b3aee032998c66bbc3abe8b5863a900c43",
     "ref_origin" : "dcos-mesos-master-abe9d1b"
```

Comment:
Whoops, I did this in the original change then I forgot to do it when I added the role tab fix.